content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md

```diff
@@ -0,0 +1,16 @@
+---
+description: ' Walk through and explanation of the LyTestTools example test. '
+title: LyTestTools Example Test Walkthrough
+linktitle: LyTestTools Example Test
+---
+
+Instructions on how to get started with an example test using the LyTestTools (LTT) testing package.
+
+## Prerequisites
+
+* Completed [Getting Started](getting-started.md)
+* A properly configured [Open 3D Engine build](/docs/user-guide/build/)
+* Open the sample test file (`Tools/LyTestTools/tests/integ/sanity_tests.py`)
+* Basic understanding of the [Pytest](https://pytest.org) module
+
+Make sure to complete all of the Prerequisite steps above. The provided example test contains in line comments to walk the user through each step. Open the the sample test and follow along for a complete walkthrough.
```

Comment:
Should this be 'in-line'?

Reply:
I believe "inline" is the correct term for this case